diff --git a/backend/config/cors.php b/backend/config/cors.php
index b571991..4088031 100644
--- a/backend/config/cors.php
+++ b/backend/config/cors.php
@@ -22,6 +22,3 @@
     'allowed_origins' => [
         'http://localhost:3000',
         'http://localhost:3001',
         'http://localhost:3002',
-        'http://127.0.0.1:3000',
-        'http://127.0.0.1:3001',
-        'http://127.0.0.1:3002',
     ],
